### PR TITLE
178 - optimize logo fetching

### PR DESCRIPTION
**Pull request by tonysjeon**

### CODE REVIEW

**Comment by awest25** [comment]
Getting `TypeError: logos.find is not a function. (In 'logos.find((team)=>team.name === matchesByDate[date][0].opponentTeam)', 'logos.find' is undefined)` not sure if it's a problem or if i just need to refresh...

File: app/components/DashboardTile.js [hunk 1]
```diff
@@ -1,44 +1,50 @@
...
-
-    fetchLogos();
-  }, [clientTeam, opponentTeam]);
+    setClientLogo(logos[clientTeam])
+    setOpponentLogo(logos[opponentTeam])
+  }, [clientTeam, opponentTeam, logos])
 
   return (
     <div className={styles.dashTilesContainer}>
+      <Helmet>
```
Comment (Fredenck) [inline app/components/DashboardTile.js:42]: is this needed? what metadata is being added?

Comment (tonysjeon) [reply app/components/DashboardTile.js]: Used to test methods to speed up the logo fetching for the tiles. Meant to remove in last commit, just updated code.

File: package.json [hunk 2]
```diff
@@ -15,6 +15,7 @@
     "next": "^14.1.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-helmet": "^6.1.0",
```
Comment (Fredenck) [inline package.json:18]: make sure to also remove this dependency as it is unused

**Review by Fredenck — APPROVED** [review]
awesome work, it's a good idea to cache the images in the browser. 

If in the future we find that rendering images is still a slow issue, we can look into memoizatino (useMemo or useCallback). Though for this scenario I don't think we need it, especially because logos are not heavy on the pixel/resolution. Plus I like to avoid complicating/unecessary changes

I'll take out the dependency and merge the changes

**Pull request merged**